pages/constitution.md

```diff
@@ -194,5 +206,7 @@ Constitutions may also include a “Provision for By-Laws”; Article or a claus
 
 * The Web Page Submissions Committee shall be comprised of the officers who will select appropriate submissions for           inclusion  on the Student Organization Web Page. 
 * The officers shall be empowered to select a member to sit on the Web Page Submissions Committee. 
-* The time and place of regular meetings may be changed by vote of the membership. 
- 
+* The time and place of regular meetings may be changed by vote of the membership.
+
```

Comment:
I saw that as well. I just assumed it was just moved around; copied and pasted else where?